Zend/tests/name_collision_01.phpt

```diff
@@ -8,4 +8,4 @@ class A { }
 
 ?>
 --EXPECTF--
-Fatal error: Cannot declare class A, because the name is already in use in %s on line %d
+Fatal error: Cannot redeclare class A (previously declared in %s:%d) in %s on line %d
```

Comment:
What do you think about shortening the error message when both names are in the same file? 

```suggestion
Fatal error: Cannot redeclare class A (previously declared on line %d) in %s on line %d
```

Reply:
And if we did, we should do the same for all message. Let's keep it as-is for now.